### PR TITLE
Fix const_err warning

### PR DESCRIPTION
**Pull request by gkoz**
Rely on glib::translate instead of a hacky one-off

### CODE REVIEW

File: src/rt.rs [hunk 1]
```diff
@@ -135,13 +135,10 @@ fn pre_init() -> bool {
...
-            b"\0" as *const u8 as *mut c_char,
-            b"--gtk-debug=misc\0" as *const u8 as *mut c_char,
-        ];
-        let mut argc = args.len() as c_int;
-        let mut argv = args.as_mut_ptr();
-        let ret = from_glib(ffi::gtk_parse_args(&mut argc, &mut argv));
+        let argv = ["", "--gtk-debug=misc"];
+        let mut argc = argv.len() as c_int;
+        let mut argv_stash = argv.to_glib_none();
+        let ret = from_glib(ffi::gtk_parse_args(&mut argc, &mut argv_stash.0));
```
Comment (EPashkin) [inline src/rt.rs]: `&mut argv.to_glib_none().0`?

**Comment by EPashkin** [comment]
:+1:

File: src/rt.rs (hunk 1 above)
Comment (gkoz) [inline src/rt.rs]: Possible but too dense magic already.

Comment (EPashkin) [inline src/rt.rs]: Ok

**Comment by GuillaumeGomez** [comment]
So no magic, ok! :D

Thanks @gkoz!

**Pull request merged**